### ISSUE TITLE
darts hub hangs after starting 

### ISSUE THREAD

**Issue by RONMAN87** [lbormann/darts-hub#8]
Hello,

unfortunately darts-hub always hangs at the “checking for update” window after starting. I have no way of getting into the program. I use a Mac M1 from 2021. I have already tried a younger version v0.12.0 with the same result. I also tried to bypass the process with the text file “my_version.txt” but unfortunately it didn’t work either. What else can I do to get the app running?

<img width="519" alt="Bildschirmfoto 2024-12-02 um 18 22 00" src="https://github.com/user-attachments/assets/3ee74f1e-ed71-4dab-9cad-1d9f6385b3ac">

Thank you very much in advance.

Best regards
Ronny

**Comment by Peschi90** [comment]
unfortunetly im not able to reproduce this behavior. Seems that the hub is not able to connect to the github repository. 
please check if you can access https://github.com/lbormann/darts-hub/releases/download from pc running darts-hub

**Comment by BaBoZz2704** [comment]
I had the same problem you have to change things in the Windows settings for proxy,
- automatically detect settings on,
- Setupscripts off
- Proxyserver using off
carefull, after eache restart it reset this settings

**Comment by Peschi90** [comment]
> I had the same problem you have to change things in the Windows settings for proxy,
> 
> * automatically detect settings on,
> * Setupscripts off
> * Proxyserver using off
>   carefull, after eache restart it reset this settings

yep, windows is a *itch, its running much better for linux but i have a lot users who are also running it on windows without problems.
Mostly Windows Defender is the problem.

**Comment by Don1589** [comment]
Same here with a raspi 5. Darts-hub Starts and „Check for updates“ appears but then it closes and nothing happens. Opening the ip for board Manager and autodarts login works with no Problems. Some people in Discord Channel have Same Problems with windows setup. What can we try that it will work again?

**Comment by Peschi90** [comment]
@Don1589 
> Same here with a raspi 5. Darts-hub Starts and „Check for updates“ appears but then it closes and nothing happens. Opening the ip for board Manager and autodarts login works with no Problems. Some people in Discord Channel have Same Problems with windows setup. What can we try that it will work again?

Thats not the same. But your behaviour is solved in the new release it was a bug in v1.1.10
